src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    String taskName;
+    boolean isDone;
```

Comment:
Perhaps the boolean isDone can be initialized to false, since when a task is first created, it is usually not completed